File: src-ts/Commands/Music/queue.ts
import { Command } from "../../Interfaces";

export const command: Command = {
  name: "queue",
  description: "Shows the queue",
  category: "Music",
  aliases: ["q"],
  run: (variables) => {
    variables.client.servers.get(variables.guild_id)?.player.queue(variables);
  }
}

File: src/bot/commands/music/queue.rs
use crate::bot::commands::{Context, Error};

/// Prints song queue
#[poise::command(
    slash_command,
    prefix_command,
    aliases("q"),
    category = "Music",
    guild_only
)]
pub async fn queue(ctx: Context<'_>) -> Result<(), Error> {
    let server = get_server!(ctx);

    server.player.print_queue(&ctx).await;

    Ok(())
}
